test/metric_learn_test.py

```diff
@@ -929,7 +929,7 @@ def test_singleton_class(self):
       X = X[[ind_0[0], ind_1[0], ind_2[0]]]
       y = y[[ind_0[0], ind_1[0], ind_2[0]]]
 
-      A = make_spd_matrix(X.shape[1], X.shape[1])
+      A = make_spd_matrix(n_dim=X.shape[1], random_state=X.shape[1])
```

Comment:
Where are you observing the failed tests? I see the CI has been passing.

Reply:
From branch master with a fresh virtual environment I got these errors:

```
FAILED test/metric_learn_test.py::TestSCML::test_iris[lda] - assert 0.255932896034129 < 0.24

FAILED test/metric_learn_test.py::TestNCA::test_one_class - TypeError: make_spd_matrix() takes 1 positional argument but 2 were given

FAILED test/metric_learn_test.py::TestNCA::test_singleton_class - TypeError: make_spd_matrix() takes 1 positional argument but 2 were given

FAILED test/test_mahalanobis_mixin.py::test_init_mahalanobis[MMC] - TypeError: make_spd_matrix() takes 1 positional argument but 2 were given

FAILED test/test_mahalanobis_mixin.py::test_init_mahalanobis[MMC_Supervised] - TypeError: make_spd_matrix() takes 1 positional argument but 2 were given
```
4 of them are due to `make_spd_matrix() ` function